symphony-bdk-examples/bdk-core-examples/src/main/java/com/symphony/bdk/examples/DatafeedExampleMain.java

```diff
@@ -0,0 +1,42 @@
+package com.symphony.bdk.examples;
+
+import com.symphony.bdk.core.SymphonyBdk;
+import com.symphony.bdk.core.api.invoker.ApiException;
+import com.symphony.bdk.core.auth.exception.AuthInitializationException;
+import com.symphony.bdk.core.auth.exception.AuthUnauthorizedException;
+import com.symphony.bdk.core.config.BdkConfigLoader;
+import com.symphony.bdk.core.config.exception.BdkConfigException;
+import com.symphony.bdk.core.config.model.BdkConfig;
+import com.symphony.bdk.core.service.datafeed.DatafeedEventListener;
+import com.symphony.bdk.core.service.datafeed.DatafeedService;
+import com.symphony.bdk.gen.api.model.V4Event;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class DatafeedExampleMain {
+
+    public static void main(String[] args) throws ApiException, BdkConfigException, AuthInitializationException, AuthUnauthorizedException {
+
+        // load configuration from classpath
+        final BdkConfig config = BdkConfigLoader.loadFromClasspath("/config.yaml");
+
+        SymphonyBdk bdk = new SymphonyBdk(config);
+        log.info("DatafeedV1: Start");
+        DatafeedService ds = bdk.datafeed();
+
+        ds.subscribe(new DatafeedEventListener() {
+            @Override
+            public void onMessageSent(V4Event event) {
+                if (event.getPayload() != null && event.getPayload().getMessageSent() != null
```

Comment:
as a user should I have to call event.getPayload().getMessageSent().getMessage() or should I be able to just retrieve event.getMessage()?

Reply:
In the legacy sdk, as far as I remember, there's several tickets asking for Initiator of an event, therefore, I think user needs more information rather than just a message payload.